trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilities.js

```diff
@@ -1,16 +1,18 @@
-const mapFacilitySnapshot = require('./mapFacilitySnapshot');
-const mapFacilityTfm = require('./mapFacilityTfm');
+const facilityMapper = require('../../facility');
 
-const mapFacilities = (facilities, dealDetails, dealTfm) => {
+/**
+ * Maps all the facilities from the database associated with a deal to their corresponding facility objects for use in TFM.
+ * In particular, these are live facility objects that update e.g. when amendments are added in TFM.
+ * This function is only used on BSS/EWCS facilities.
```

Comment:
We can remove this line 😎

Reply:
This function is only used on BSS/EWCS facilities (there's a seperate `mapGefFacilities` file) so possibly helpful to keep?

Reply:
(although it would now work on both types.. maybe I'll remove it)

Reply:
Removed!